### PR TITLE
Request to merge Add user

### PR DESCRIPTION
**Pull request by Kingstalux**
- Remove current_user method in ApplicationController.
- [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
- Can register a new user.
- User logs in with a combination of email and password.
- Hashed passwords are stored in the database.
- Can reset password.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (hiromataba) [inline app/models/user.rb]: - [x] Please enable `:confirmable`   in the user model for the email confirmation

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Kingstalux :wave: :wave:  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_sports: 
- Nice PR title and description :heavy_check_mark: 
- Professional README file :heavy_check_mark: 
- Your linters are passing :heavy_check_mark: 

## Required Changes ♻️

- [x] When i am trying  to create the database I am getting the error below :point_down:  please fix that

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] [OPTIONAL] I think it would be better to update your ruby version, you can have a look at [this documentation](https://docs.bitnami.com/ibm/infrastructure/ruby/get-started/understand-rbenv/) 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (Kingstalux) [reply app/models/user.rb]: Hello @hiromataba, thanks for the review. I have added the email confirmation. As for the first issue you raised, it is probably due to the fact that we have different ruby versions.

**Review by dasileker — APPROVED** [review]
### **Hi @Kingstalux,** 

### **Congratulations 👌👌**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**